drivers/volume/portworx/portworx.go

```diff
@@ -3612,6 +3627,26 @@ func (d *portworx) UpdateStorageClusterImage(imageName string) error {
 
 }
 
+func (d *portworx) GetPXStorageCluster() (*v1.StorageCluster, error) {
+	pxOps, err := pxOperator.ListStorageClusters(schedops.PXNamespace)
+	if err != nil {
+		er := fmt.Errorf("Error getting Storage Clusters list, Err: %v", err.Error())
```

Comment:
you can still use `err` here instead of `er` since it's a local variable because you are using `:=` in line 3633